packages/components/src/toggle-group-control/toggle-group-control/as-button-group.tsx

```diff
@@ -92,7 +93,7 @@ function UnforwardedToggleGroupControlAsButtonGroup(
 					containerWidth={ sizes.width }
 					isAdaptiveWidth={ isAdaptiveWidth }
 				/>
-				{ children }
+				<HStack spacing={ 1 }>{ children }</HStack>
```

Comment:
Spacing was a bit tight to begin with in the `isDeselectable` ToggleGroupControl, but it becomes very noticeable when multiple buttons are pressed. Added some breathing room.

<img width="121" alt="multiple buttons pressed" src="https://user-images.githubusercontent.com/555336/200401445-d1299bbb-7f53-4998-831a-ec2926dcd42a.png">

Reply:
Gap [increased](https://github.com/WordPress/gutenberg/pull/45128#issuecomment-1306856488) to 8px:

<img width="150" alt="buttons with 8px gap" src="https://user-images.githubusercontent.com/555336/200643121-2a42379a-43dc-45a2-bfd2-46f4dff30c85.png">